api/v1beta1/definition.proto

```diff
@@ -4,12 +4,15 @@ option go_package = "github.co/aquasecurity/tracee/api/v1beta1";
 
 package tracee.v1beta1;
 
+import "api/v1beta1/threat.proto";
+
 message EventDefinition {
 	int32 id = 1;
 	string name = 2;
 	Version version= 3;
 	string description = 4 ;
 	repeated string tags = 5;
+	optional Threat threat = 6;
```

Comment:
nit: I think that this should be called properties to align with the internal event definition concept, or vice versa, your call.

Reply:
hmm, I think Threat is best, because we already have properties on it.